devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -58,17 +58,24 @@ metadata {
 }
 
 def updated() {
-    stopTimer()
-    startTimer()
+	log.debug "updated()"
+    state.gsensor = 0
+    def thedetectTime = (detectTime ? detectTime as int : 2) * 1
+    def updatecmds = zigbee.writeAttribute(0x0000, 0x0000, 0x20, thedetectTime, [mfgCode: 0x120D]) 
+    log.debug "Updatecmds:  ${updatecmds}"
+	return response(updatecmds)
 }
 
 def installed() {
-    // Arrival sensors only goes OFFLINE when Hub is off
+	log.debug "installed()"
+	// Arrival sensors only goes OFFLINE when Hub is off
     sendEvent(name: "DeviceWatch-Enroll", value: JsonOutput.toJson([protocol: "zigbee", scheme:"untracked"]), displayed: false)
 }
 
 def configure() {
-    def cmds = zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020) + zigbee.batteryConfig(20, 20, 0x01)
+    def cmds = zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020) + zigbee.batteryConfig(60, 60, 0x01) +   //30  //3600 -> 1hour
+                zigbee.readAttribute(zigbee.ONOFF_CLUSTER, 0x0000) + zigbee.onOffConfig() //+ //zigbee.configureReporting(0x0006, 0x0000, DataType.BOOLEAN, 5, 5, 0x01)
```

Comment:
If Your device need some extra configuration please add "if" statements to handle it (that way it will not be sent to other devices that are supported by this DTH).

Reply:
OK! I will add "if" statements.